app/src/main/res/layout/layout_listitem.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
clean this file remove extra spaces in between.
1 space is enough.

Reply:
done